toolchain/private/defs.bzl

```diff
@@ -121,16 +121,6 @@ def _target_windows(gocpu, zigcpu):
 def _target_linux_gnu(gocpu, zigcpu, glibc_version):
     glibc_suffix = "gnu.{}".format(glibc_version)
 
-    compiler_extra_includes = []
```

Comment:
look for `compiler_extra_includes` and `linker_extra_includes` in this repository -- there may be more dead code to be removed.